schema/omega-services.async.api.yaml

```diff
@@ -414,6 +545,222 @@ components:
           items:
             $ref: '#/components/schemas/GeneralDatedDescription'
 
+    ListAgentParameters:
+      type: object
+      properties:
+        type:
+          $ref: '#/components/schemas/AgentTypes'
+          description: Limits the list to one-or-more types of Agent. If omitted, all types of agents are included in the list.
+        authority-file:
+          $ref: '#/components/schemas/AuthorityFile'
+          description: Limits the list of Agent(s) to only those that are (or are not) also Authority Files. If omitted both authorative and non-authorative Agents are included in the list.
+        depository:
+          $ref: '#/components/schemas/Depository'
+          description: Limits the list of Agent(s) to only those that are (or are not) also a depository. If omitted both depository and non-depository Agents are included in the list.
+        version-timestamp:
+          $ref: '#/components/schemas/VersionTimestamp'
+          description: Limits the list of Agent(s) to only those that have a version timestamp that is greated than or equal to this timestamp.
+          default: latest
```

Comment:
It doesn't say that any of these properties are required. This implies that the payload is optional. Is that the intention?

Reply:
Yes, exactly. These are parameters for filtering the list of Agents. If there are no parameters needed, it would seem redundant to send a payload that is empty.

Reply:
Does that mean we can consider the following to be the default values:
```
{
    "type" : ["Collective Agent", "Corporate Body", "Family", "Hardware Agent", "Person", "Software Agent" ],
    "authority-file" : false,
    "depository" : false,
    "version-timestamp" : "latest"
}
```

Reply:
Does the `*` represent any value? So any type of agent and we are concerned whether or not they are authority files or depositories?

Reply:
@rwalpole Exactly!